invenio_records_resources/factories/factory.py

```diff
@@ -182,8 +198,7 @@ def create_service_class(self):
         if self.service_components:
             config_cls_attributes.update(
                 {
-                    "components": RecordServiceConfig.components
-                    + self.service_components
+                    "components": self.service_components
```

Comment:
If the user wants to get the parent ones s/he can inherit. Otherwise, there is no way of e.g. not get the `MetadataComponent`.